packages/integration-tests/test/depositTest.js

```diff
@@ -117,7 +117,6 @@ describe('depositTest.js (ci-enabled)', function () {
       // THe account should have one utxo on the child chain
       const utxos = await childChain.getUtxos(aliceAccount.address)
       assert.equal(utxos.length, 1)
-      assert.hasAllKeys(utxos[0], ['utxo_pos', 'txindex', 'owner', 'oindex', 'currency', 'blknum', 'amount', 'creating_txhash', 'spending_txhash', 'otype'])
```

Comment:
instead of removing the assert completely, does it make sense to change it to `assert.containsAllKeys()`?

same on line 184

Reply:
removed it because these tests doesn't rely on all the keys in the response, only that the amount and currency is correct. so didn't want this test to be responsible for testing the shape of the response, but just the functionality of depositing